optaplanner-quickstarts/pom.xml

```diff
@@ -23,6 +23,7 @@
   <url>https://www.optaplanner.org</url>
 
   <modules>
+    <module>quarkus-school-timetabling</module>
```

Comment:
Quarkus should not be part of default build on branch 7.x
Better to remove it completely and keep it only on master branch, wdyt @ge0ffrey @triceo @rsynek ?